warehouse/models/mart/ntd_annual_reporting/fct_2023_contractual_relationships.sql

```diff
@@ -3,9 +3,25 @@ WITH staging_contractual_relationships AS (
     FROM {{ ref('stg_ntd__2023_contractual_relationships') }}
 ),
 
+current_dim_organizations AS (
+    SELECT
+        ntd_id,
+        caltrans_district
+    FROM {{ ref('dim_organizations') }}
+    WHERE _is_current
```

Comment:
This leaves open the potential for drift in the event that Caltrans district boundaries change or the transit agency HQ changes in the future. There could be a situation where the 2023 mapping should have been a different Caltrans District based upon past data. Not sure how much effort we want to put into preventing this edge case since either of those changes don't occur very often.